docs/guides/earning-cred.md

```diff
@@ -1,42 +1,41 @@
 ---
 title: 💧 Earning Cred
-description: How to get Cred from your contributions.
+description: How you get Cred from your contributions.
 ---
 
-As soon as you interact with SourceCred, you will be part of the Contribution Graph (i.e. a network of connected contributions). Being part of this Graph is like having your own pond now that holds [Cred]. Cred will flow in&out in different ways, just like water in a pond. 
+As soon as you interact with SourceCred, you will be part of the Contribution Graph (i.e. a network of connected contributions). Being part of this graph is like having your own pond that holds [Cred]. Cred will flow in and out in different ways, just like water in a pond. 
 
 
 
 ### 💧 Cred Minting
-The community decides to Mint some Cred for you, e.g. because they found something valuable. This Cred just magically appears out of nowhere in your direction. If we compare Cred to water, minting Cred would be like rain - it would fall from the sky, new water collected in your pond !
+If community members find any of your contributions valuable, they **Mint**, or in other words, create, Cred, and flow it to you. This Cred just magically appears out of nowhere in your direction. If we compare Cred to water, minting Cred would be like rain - it would fall from the sky, new water collected in your pond!
 
-The types of contributions & plugins are likely to increase. But for now, small is beautiful. Your contributions are tracked across the three active plugins in the following ways:
+The types of contributions and plugins are likely to increase. But for now, small is beautiful. Your contributions are tracked across the these active plugins in the following ways:
 
 
 | 💬 [Discord] | 🧵 [Discourse] | 🦠 [Github]| 🥇 [Initiatives]
 | -- | -- | -- | -- |
-|someone gives your post : <img width="20" alt="SourceCred" src='https://sourcecred.io/img/favicon.png' /> or :100: or :+1:| someone gives your post/reply a :heart: | You open a pull request  | you make a specific task approved by the community
+|someone gives your post any emoji : <img width="20" alt="SourceCred" src='https://sourcecred.io/img/favicon.png' /> :100: :+1:| someone gives your post/reply a :heart: | You open a pull request  | you accomplish a specific task approved by the community
 
+### 🔥 Radiation
+Since Cred can be created from nothing by community members, there must be a balance so that Cred maintains its worth. Our system is called **Cred Radiation**, and is similar to how water evaporates from all water sources. For every node on the Contribution Graph, a fixed proportion of its Cred radiates away, and that fixed proportion is called alpha.
 
+### 🏄🏾 Cred Flowing
```

Comment:
Like this wording as a high-level picture of what's going on. Perhaps move up to beginning of article?

Reply:
I've been playing around a lot with the order of the sections. It's tough bc they all overlap and to understand one you need to understand them all, to some degree. I think I have it good now, happy to hear further suggestions on the new version